src/webhook/validation/oneagent_test.go

```diff
@@ -290,3 +295,141 @@ func TestImageFieldSetWithoutCSIFlag(t *testing.T) {
 	})
 
 }
+
+func TestOneAgentVolumeStorageReadOnlyModeConflict(t *testing.T) {
```

Comment:
please rework these tests so they align with the sample in the contribution guidelines

Reply:
That just leads to lots of code duplication, long test methods, obfuscation of the tiny differences in the tests and doesn't provide any benefit to code quality whatsoever. One thing to rework is the test naming, Testnames should be written in the code (not generated as of now) to make them easier to find when failing.

Reply:
Also test maintenance get's potentially more cumbersome with lot's of duplicated code.